src/components/StaticBox.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { ReactNode, CSSProperties } from 'react'
+import { Paragraph, Separator, Text, XStack, YStack } from 'tamagui'
+import { styled } from 'tamagui'
+import { Image } from 'tamagui'
+import MyResponsiveLine from './StandardLineChart'
+
+interface IconProps {
+  source: string
+  width?: number
+  height?: number
+}
+
+const Icon: React.FC<IconProps> = ({ source, width = 16, height = 16, ...props }) => {
+  return <Image {...props} source={{ uri: source }} width={width} height={height} />
+}
+
+interface IconTextProps {
+  icon: string
+  text: string
+  fontWeight?: string
+}
+
+const IconText: React.FC<IconTextProps> = ({ icon, text, ...props }) => {
+  return (
+    <XStack
+      style={{
+        alignItems: 'center',
+      }}
+      space={'$2'}
+    >
+      <Icon source={icon} width={16} height={16} />
+      <Paragraph {...props} color={'#000000'}>
+        {text}
+      </Paragraph>
+    </XStack>
+  )
+}
+const DivComponent: React.FC = props => <div {...props} />
+const ShadowBox = styled(DivComponent, {
```

Comment:
You can remove DivComponent and ShadowBox because we already have ShadowBox component